test/built-ins/ArrayBuffer/prototype/detached/this-is-sharedarraybuffer-resizable.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2023 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-get-arraybuffer.prototype.detached
+description: Throws a TypeError exception when `this` is a resizable SharedArrayBuffer
+info: |
+  get ArrayBuffer.prototype.detached
+
+  1. Let O be the this value.
+  2. Perform ? RequireInternalSlot(O, [[ArrayBufferData]]).
+  3. If IsSharedArrayBuffer(O) is true, throw a TypeError exception.
+  [...]
+features: [SharedArrayBuffer, resizable-arraybuffer, ArrayBuffer, arraybuffer-detached]
+---*/
+
+var detached = Object.getOwnPropertyDescriptor(
+  ArrayBuffer.prototype, "detached"
+);
+
+var getter = detached.get;
+var sab = new SharedArrayBuffer(4);
+
+assert.sameValue(typeof getter, "function");
+
+Object.defineProperties(sab, { maxByteLength: maxByteLength });
```

Comment:
This seems like a mistake--`maxByteLength` isn't defined, and defining any property won't make the ArrayBuffer resizable.